### PR TITLE
[docs] Improve MatrixTable.entries docs

### PR DESCRIPTION
**Pull request by tpoterba**

### CODE REVIEW

**Comment by johnc1231** [comment]
Failing `make_docs`

**Comment by johnc1231** [comment]
Bump, failing a docs build

**Comment by tpoterba** [comment]
thanks for the reminder!

File: hail/python/hail/matrixtable.py [hunk 1]
```diff
@@ -2703,6 +2703,21 @@ def entries(self) -> Table:
 
         >>> entries_table = dataset.entries()
 
+        Notes
+        -----
+        The coordinate table representation of the source matrix table contains
+        one row for each **non-filtered** entry of the matrix -- if a matrix table
+        has no filtered entries nad contains N rows and M columns, the table will contain
```
Comment (johnc1231) [inline hail/python/hail/matrixtable.py:2710]: `nad` -> `and`

**Review by johnc1231 — CHANGES_REQUESTED** [review]
One typo, otherwise looks good.

**Comment by johnc1231** [comment]
Bump. Tiny typo and will be good to go.

**Review by johnc1231 — APPROVED** [review]

**Pull request merged**